components/perl/Tk/patches/01-load-64-bit-X11.patch

```diff
@@ -0,0 +1,16 @@
+Since we build 64 bit we need to load 64 bit libX11.
+See https://github.com/eserte/perl-tk/issues/99
+
+--- Tk-804.036/Tk.pm.orig
++++ Tk-804.036/Tk.pm
+@@ -90,8 +90,8 @@
+ my $boot_time = timeofday();
+ 
+ # This is a workround for Solaris X11 locale handling
+-Preload(DynaLoader::dl_findfile('-L/usr/openwin/lib','-lX11'))
+-  if (NeedPreload() && -d '/usr/openwin/lib');
++Preload(DynaLoader::dl_findfile('-L/usr/openwin/lib/64','-lX11'))
```

Comment:
What's questionable here is that `/usr/openwin` is a symlink to `/usr/X11`, and `/usr/X11` is mostly symlinks pointing at files in the usual subdirectories of `/usr`, all left in the filesystem for reasons of backwards compatibility across two migrations of names.

If the code needs to load `-lX11`, I'd think it should get it from the default library path..

```
$ realpath /usr/openwin/lib/64/libX11.so
/usr/lib/amd64/libX11.so.4.0.0
```